A few improvements in code and CSS along with some config parameters.

u-quark opened the pull request:
Just a few improvements that I found useful:
- Silence some php warnings.
- Make better CSS alignment in totals.
- Add config parameter: Control refresh rate.
- Add config parameter: Add extra external CSS.

u-quark committed:
Make php not complain about filter_sort_descending

u-quark committed:
Fix 'Undefined index: url'

u-quark committed:
Fix undefined index: objects

u-quark committed:
Add config parameter for refresh rate

u-quark committed:
Better CSS alignment on the right side

u-quark committed:
Add config option to add extra css for custom skin

u-quark committed:
Don't use short php tags

lozzd commented:
Hey there!
Would you mind adding some logic that defaults to 20 seconds if the ```$refresh_every_ms``` variable isn't set? (which would happen if people upgrade and don't spot the new config param). 
Thanks!

stevenschlansker commented:
Fixes #54

u-quark committed:
Set refresh rate to default if not set in config

This is done to smooth the transition of introducing the new
configuration parameter when updating with an older config.

u-quark commented:
@lozzd Yeap, it makes sense to do it this way.

Thank you.

lozzd commented:
Thanks!

u-quark commented:
Cheers!
